src/transformStateWithClones.js

```diff
@@ -7,7 +7,30 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const allActions = []; // final array
```

Comment:
this one should not be called `allActions`, it's more like a history of states, name it accordingly